### DISCUSSION TITLE
Compress the model generated by dpgen

### DISCUSSION THREAD

**Discussion by jic198** [deepmodeling/dpgen#1170] [General]
Hello. I noticed that when I used the model generated by dpgen to run lammps, its speed was much slower than the one generated by deepmd-kit. I typically compressed the model to speed up the MD calculations. Is there a way to compress the model generated by dpgen? I tried `dp compress -i graph.pb -o graph-compress.pb` but got a following error:
`AssertionError: Model compression error: descriptor resnet_dt must be false!` Could you help me out? Thank you!

**Comment by chunweizhu** [comment]
Maybe, you need to change the parameter `resnet_dt: false` in the param.json file and train the model once again, i.e. without using the residual network.
